protos/channel_responses.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+/* Following chromium style (e.g. sync_pb) is the usage for the sync protobuf
+*/
+package publishers_pb;
+
+message SocialLinks {
+  string youtube = 1;
+  string twitter = 2;
+  string twitch = 3;
+}
+
+message SiteBannerDetails {
+  string title = 1;
+  string description = 2;
+  string background_url = 3;
+  string logo_url = 4;
+  repeated int32 donation_amounts = 5;
+  SocialLinks social_links = 6;
+}
+
+enum WalletConnectedState {
+  NO_VERIFICATION = 0;
+  UPHOLD_ACCOUNT_NO_KYC = 1;
+  UPHOLD_ACCOUNT_KYC = 2;
+  /* (Albert Wang) These might be used */
+  PAYPAL_ACCOUNT_NO_KYC = 3;
```

Comment:
Will these be included in the list? We don't currently support "paypal" as such on the client side.

Reply:
We likely will include Stripe. I personally don't mind since these fields are optional, but I'll remove to keep it clean and add once it is necessary.

Reply:
Since the client isn't going to know how to deal with these values, it would be fine to take them out (and would leave more options open for later).